### PR TITLE
GH-884: ROC The Space Fixes

### PR DESCRIPTION
**Pull request by yuenyauu**
- Increased tooltip sizes
- Increased font size for Crocodile Centre
- Adjusted tooltip positioning

### CODE REVIEW

**Review by tamarayustian — APPROVED** [review]
interesting that you do negative. why not use bottom (like opposite direction) instead of negative with top? approved anyway tho

File: ui/src/components/ripple-out/RippleOutSpaceContainer.js [hunk 1]
```diff
@@ -149,13 +149,13 @@ const RippleOutSpaceContainer = () => {
               placement="top"
             >
               <Box
-                top={['-6vh', '-14vh']}
-                left={['-30vw', '-18vw']}
+                top={['-6.5vh', '-17vh']}
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/RippleOutSpaceContainer.js:152]: image
the dots now look misaligned to me? the building blocks dot is now out of the room and the office dot is covering the text

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

File: ui/src/components/ripple-out/RippleOutSpaceContainer.js (hunk 1 above)
Comment (yuenyauu) [reply ui/src/components/ripple-out/RippleOutSpaceContainer.js:152]: image
looks good on mine though lolol

Comment (yuenyauu) [reply ui/src/components/ripple-out/RippleOutSpaceContainer.js:152]: i guess i could make the buttons slightly smaller

**Review by tamarayustian — COMMENTED** [review]
image
since you're on this file, can you also remove these unnecessary imports? thanks!

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**